### DISCUSSION TITLE
Add/Change a Script Shortcut

### DISCUSSION THREAD

**Discussion by johnlindquist** [johnlindquist/kit#532] [Docs]
<meta path="kit/change-shortcut">
      
# Add/Change a Script Shortcut

This list all the scripts and allows you to add a shortcut to it.

You can manually add shortcuts to scripts like so:

```js
// Shortcut: cmd option g
```

This menu will manage that for you

**Comment by iamjoel** [comment]
Script Shortcut is not work in Script-Kit-macOS-3.14.6-arm64
